src/models/components.d.ts

```diff
@@ -0,0 +1,64 @@
+import { SvgIconComponent } from "@mui/icons-material";
+
+/*=============================================
+=            // 1,  factSearchLists            =
+=============================================*/
+
+export interface listItemTypes {
+  name: string;
+  path: string;
+  image: string;
+}
+export interface GroupListsTypes {
+  listItems: Array<listItemTypes>;
+  groupTitle: string;
+}
+interface extendsProps {
+  onSelectItem: (item: listItemTypes) => item;
+  onViewMore?: () => void;
+}
+
+export type FactSearchListsPropsTypes = (
```

Comment:
singular vs plural usage in naming convention, please fix

Reply:
See all other places where you have used it. For example, `FactlListItemPropTypes` should `FactlListItemPropType`